pkgs/tools/misc/ollama/default.nix

```
@@ -55,13 +56,20 @@ let
   };
 
 
-  validAccel = lib.assertOneOf "ollama.acceleration" acceleration [ null "rocm" "cuda" ];
-
+  validateAccel = lib.assertOneOf "ollama.acceleration" acceleration [ null "rocm" "cuda" ];
```

```suggestion
  validAccelerator = assert lib.assertOneOf "ollama.acceleration" acceleration [ null false "rocm" "cuda" ]; acceleration;
```

Then you don't have to `validateAccel &&` below

---

Thank you; I was unaware of nix's `assert` language feature.